examples/ss3_template_files/starter.ss

```diff
@@ -1,44 +1,66 @@
-#V3.30.19.01;_safe;_compile_date:_Apr 15 2022;_Stock_Synthesis_by_Richard_Methot_(NOAA)_using_ADMB_12.3
+#V3.30.20.00;_safe;_compile_date:_Sep 30 2022;_Stock_Synthesis_by_Richard_Methot_(NOAA)_using_ADMB_13.0
 #_Stock_Synthesis_is_a_work_of_the_U.S._Government_and_is_not_subject_to_copyright_protection_in_the_United_States.
 #_Foreign_copyrights_may_apply._See_copyright.txt_for_more_information.
 #_User_support_available_at:NMFS.Stock.Synthesis@noaa.gov
 #_User_info_available_at:https://vlab.noaa.gov/group/stock-synthesis
 #_Source_code_at:_https://github.com/nmfs-stock-synthesis/stock-synthesis
 
-2021_model_species.dat
-2021_model_species.ctl
+#C starter comment here
+# PFMC: #############################################################
+# PFMC: .[a-z].ss ensures data and control file are first.
+# PFMC: #############################################################
+.data.ss
+.control.ss
 0 # 0=use init values in control file; 1=use ss.par
-1 # run display detail (0 = none, 1 = one brief line of display for each iterations, 2=fuller display per iteration)
+# PFMC: #############################################################
+# PFMC: Console and file output is verbose by default.
+# PFMC: echoinput.sso and parmtrace.sso are useful for debugging.
+# PFMC: Turn off output to run models faster.
+# PFMC: #############################################################
+2 # run display detail (0,1,2)
```

Comment:
I did a comparison of the impact of the "run display detail" options using the [simple_small](https://github.com/r4ss/r4ss/tree/main/inst/extdata/simple_small) model in {r4ss}. Results are attached. Here's a brief summary:

- Detail = 0: output is 47 lines, easy to focus on message about parameter near bound, 
- Detail = 1: output is 295 lines, most of which is 1 line per each of the 248 iterations showing phase, total likelihood, and spawning biomass at start and end
- Detail = 2: output is 3519 lines, 3224 of which is 13 lines of likelihood by data type for each of the 248 iterations. Warning about parameter near bound  is on line 1723, so hard to find if you're not searching for it.

My conclusion is that run display detail = 2 should only be used if the model crashes or behaves strangely and you're trying to figure out why. For me the choice of detail = 0 vs 1 depends on whether I'm going to watch the output scroll by. If so, I would use 1 so I can see how fast the likelihood is improving and whether Spbio values are reasonable while it's optimizing.

[console.output_detail0.txt](https://github.com/pfmc-assessments/PEPtools/files/9779223/console.output_detail0.txt)
[console.output_detail1.txt](https://github.com/pfmc-assessments/PEPtools/files/9779224/console.output_detail1.txt)
[console.output_detail2.txt](https://github.com/pfmc-assessments/PEPtools/files/9779225/console.output_detail2.txt)

Reply:
Thanks @iantaylor-NOAA I changed it to 0 because I think that the default mode would be to run a lot of models and look at the output that is saved to files rather than watching the console output for every model that is run. I also think that your point about it being able to see what parameters are near a bound from the smaller output is important.